servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder-arm/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure.spring</groupId>
+    <artifactId>azure-spring-boot-samples</artifactId>
+    <version>1.0.0</version>
+    <relativePath>../../../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>spring-cloud-azure-sample-stream-binder-servicebus-queue-arm</artifactId>
+  <version>1.0.0</version>
+  <packaging>jar</packaging>
+
+  <name>Communicate to a Single Service Bus Queue via Spring Cloud Stream By Azure Resource Manager Authentication</name>
+
+
+  <dependencies>
+    <dependency>
+      <groupId>com.azure.spring</groupId>
+      <artifactId>spring-cloud-azure-resourcemanager</artifactId>
+    </dependency>
```

Comment:
we don't need this, the `spring-cloud-azure-stream-binder-servicebus` has already depended on it IIRC.